tests/pl/homeassistant_HassGetCurrentTime.yaml

```diff
@@ -0,0 +1,21 @@
+language: pl
+tests:
+  - sentences:
+      - "jaką mamy godzinę"
+      - "jaką mamy teraz godzinę"
+      - "jaką mamy obecnie godzinę"
+      - "jaka godzina"
+      - "jaka jest godzina"
+      - "jaka teraz godzina"
+      - "jaka jest teraz godzina"
+      - "jaka obecnie godzina"
+      - "jaka jest obecnie godzina"
+      - "która godzina"
+      - "która jest godzina"
+      - "która jest teraz godzina"
+      - "podaj mi godzinę"
+      - "podaj mi obcną godzinę"
+      - "podaj mi aktualną godzinę"
+    intent:
+      name: HassGetCurrentTime
+    response: "Jest godzina pierwsza dwa"
```

Comment:
We cannot use this placeholder because Polish voice synthesizers read the hours incorrectly, so I decided to transform it so that they are pronounced correctly.